packages/strapi-provider-email-mailgun/README.md

```diff
@@ -46,6 +46,7 @@ module.exports = ({ env }) => ({
     providerOptions: {
       apiKey: env('MAILGUN_API_KEY'),
       domain: env('MAILGUN_DOMAIN'), //Required if you have an account with multiple domains
+      host: env('MAILGUN_HOST') // If you are using EU servers, you need to specify the host. api.eu.mailgun.net for EU Servers (or api.mailgun.net for US servers)
```

Comment:
If this is optional it needs to have the default value in the env function. 

`env('MAILGUN_HOST', 'whatever the default is')`